tensorrt_llm/_torch/model_config.py

```
@@ -412,16 +412,23 @@ def from_pretrained(cls,
-            model_dir = Path(
-                transformers.utils.hub.cached_file(checkpoint_dir,
-                                                   'config.json')).parent
+                # Find the cache path by looking for the config.json file which should be in all
+                # huggingface models
+                model_dir = Path(
+                    transformers.utils.hub.cached_file(checkpoint_dir,
+                                                       'config.json')).parent
+            else:
+                raise ValueError(
+                    "checkpoint_dir is None. Cannot load model config without a valid checkpoint directory."
+                )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)